node/src/accountant/mod.rs

```diff
@@ -129,6 +133,13 @@ pub struct SentPayables {
     pub response_skeleton_opt: Option<ResponseSkeleton>,
 }
 
+#[derive(Debug, Message, PartialEq, Eq)]
+pub struct AvailableBalancesAndQualifiedPayables {
+    pub accounts: Vec<PayableAccount>,
+    pub consuming_wallet_balances: WalletBalances,
```

Comment:
I agree that my name isn't pretty but that doesn't change the fact that it does refer to two balances of our consuming wallet. 

The gas is as important as the tokens. If you don't have enough gas, the payment will fail. No excuse...
In that sense I put both balances at the same level. 

Suggested renaming: 

Instead of `consuming_wallet_balances` `consuming_balances` should work as we never can know other than our consuming wallet. 

`WalletBalance` should be renamed to `ConsumingWalletBalances`.

`AvailableBalanceAndQualifiedPayables` would work better as `ConsumingBalancesAndQualifiedPayables`.

From here on everything would be abide by this law :point_up: .

Reply:
I'd appreciate, that you're doing the effort of renames.

Yes, the names that you provided are easier to understand.